src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/source_filters_table/__tests__/source_filters_table.test.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { SourceFiltersTable } from '../source_filters_table';
+
+jest.mock('@elastic/eui', () => ({
+  EuiButton: 'eui-button',
+  EuiTableOfRecords: 'eui-table-of-records',
+  EuiTitle: 'eui-title',
+  EuiText: 'eui-text',
+  EuiButton: 'eui-button',
+  EuiHorizontalRule: 'eui-horizontal-rule',
+  EuiSpacer: 'eui-spacer',
+  EuiCallOut: 'eui-call-out',
+  EuiLink: 'eui-link',
+  EuiOverlayMask: 'eui-overlay-mask',
+  EuiConfirmModal: 'eui-confirm-modal',
+  EuiLoadingSpinner: 'eui-loading-spinner',
+  Comparators: {
+    property: () => {},
+    default: () => {},
+  },
+}));
+jest.mock('../components/header', () => ({ Header: 'header' }));
+jest.mock('../components/table', () => ({
+  // Note: this seems to fix React complaining about non lowercase attributes
+  Table: () => {
+    return 'table';
+  }
+}));
+
```

Comment:
I'm not a huge fan of all this mocking.  Seems like it will take some maintenance to keep this matching the UI components you are using, and then you miss anything that breaks as a result of the EUI stuff changing.